request.c

```diff
@@ -513,6 +513,8 @@ void getForwardDestinations(char *client_message, int *sock)
 		qsort(temparray, counters.forwarded+1, sizeof(int[2]), cmpdesc);
 	}
 
+	totalqueries = counters.forwardedqueries + counters.cached + counters.blocked;
```

Comment:
I compute this outside of the loop to avoid re-computing the very same number over and over again